templates/repo/branch/list.tmpl

```diff
@@ -76,14 +79,20 @@
 							<tr>
 								<td class="eight wide">
 								{{if .DBBranch.IsDeleted}}
-									<s><a href="{{$.RepoLink}}/src/branch/{{PathEscapeSegments .DBBranch.Name}}">{{.DBBranch.Name}}</a></s>
+									<div class="gt-df gt-ac">
+										<a class="gt-ellipsis" href="{{$.RepoLink}}/src/branch/{{PathEscapeSegments .DBBranch.Name}}">{{.DBBranch.Name}}</a>
+										<button class="btn interact-fg gt-p-3" data-clipboard-text="{{.DBBranch.Name}}">{{svg "octicon-copy" 14}}</button>
+									</div>
```

Comment:
You can probably avoid the extra wrapper via `gt-dib gt-ac` on the `<a>`.

Reply:
Not possible
<img width="1371" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/1d0bafa3-24a3-4787-93b9-aadf1cd8ae29">

we want the whole flex-box div item to be in centered alinged.
And if we apply `gt-ac`

<img width="1225" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/b71f1c3a-18c1-4a5e-82f1-a0b2e4cf4579">

The branch anchor tag is perfectly centered.